page 20570
/bbs/Gossiping/M.1495290504.A.467.html download to ptt/Gossiping/20570_0.txt, time: 6.770000
/bbs/Gossiping/M.1495290561.A.D86.html download to ptt/Gossiping/20570_1.txt, time: 5.080000
/bbs/Gossiping/M.1495290564.A.734.html download to ptt/Gossiping/20570_2.txt, time: 2.860000
/bbs/Gossiping/M.1495290605.A.466.html download to ptt/Gossiping/20570_3.txt, time: 5.040000
/bbs/Gossiping/M.1495290648.A.6FB.html download to ptt/Gossiping/20570_4.txt, time: 4.790000
/bbs/Gossiping/M.1495290654.A.1E9.html download to ptt/Gossiping/20570_5.txt, time: 4.210000
/bbs/Gossiping/M.1495290661.A.CF9.html download to ptt/Gossiping/20570_6.txt, time: 4.370000
/bbs/Gossiping/M.1495290667.A.FCC.html download to ptt/Gossiping/20570_7.txt, time: 7.290000
/bbs/Gossiping/M.1495290675.A.D2E.html download to ptt/Gossiping/20570_8.txt, time: 5.550000
/bbs/Gossiping/M.1495290708.A.251.html download to ptt/Gossiping/20570_9.txt, time: 4.330000
/bbs/Gossiping/M.1495290731.A.4A2.html download to ptt/Gossiping/20570_10.txt, time: 4.930000
/bbs/Gossiping/M.1495290735.A.923.html download to ptt/Gossiping/20570_11.txt, time: 2.480000
/bbs/Gossiping/M.1495290763.A.DB9.html download to ptt/Gossiping/20570_12.txt, time: 3.590000
/bbs/Gossiping/M.1495290776.A.04C.html download to ptt/Gossiping/20570_13.txt, time: 6.360000
/bbs/Gossiping/M.1495290777.A.3C2.html download to ptt/Gossiping/20570_14.txt, time: 5.930000
/bbs/Gossiping/M.1495290810.A.2D4.html download to ptt/Gossiping/20570_15.txt, time: 5.350000
/bbs/Gossiping/M.1495290830.A.F48.html download to ptt/Gossiping/20570_16.txt, time: 4.380000
/bbs/Gossiping/M.1495290906.A.B4A.html download to ptt/Gossiping/20570_17.txt, time: 5.040000
/bbs/Gossiping/M.1495290908.A.76F.html download to ptt/Gossiping/20570_18.txt, time: 4.920000
/bbs/Gossiping/M.1495290945.A.CA0.html download to ptt/Gossiping/20570_19.txt, time: 5.710000
page 20569
/bbs/Gossiping/M.1495290127.A.0B6.html download to ptt/Gossiping/20569_0.txt, time: 3.870000
/bbs/Gossiping/M.1495290134.A.2EE.html download to ptt/Gossiping/20569_1.txt, time: 4.870000
/bbs/Gossiping/M.1495290138.A.BF5.html download to ptt/Gossiping/20569_2.txt, time: 6.500000
/bbs/Gossiping/M.1495290150.A.E7A.html download to ptt/Gossiping/20569_3.txt, time: 4.410000
/bbs/Gossiping/M.1495290211.A.DC0.html download to ptt/Gossiping/20569_4.txt, time: 4.340000
/bbs/Gossiping/M.1495290233.A.1FE.html download to ptt/Gossiping/20569_5.txt, time: 5.490000
/bbs/Gossiping/M.1495290274.A.292.html download to ptt/Gossiping/20569_6.txt, time: 3.570000
/bbs/Gossiping/M.1495290289.A.FC6.html download to ptt/Gossiping/20569_7.txt, time: 5.000000
/bbs/Gossiping/M.1495290292.A.9D8.html download to ptt/Gossiping/20569_8.txt, time: 3.890000
/bbs/Gossiping/M.1495290313.A.9BF.html download to ptt/Gossiping/20569_9.txt, time: 6.400000
/bbs/Gossiping/M.1495290317.A.556.html download to ptt/Gossiping/20569_10.txt, time: 4.060000
/bbs/Gossiping/M.1495290323.A.487.html download to ptt/Gossiping/20569_11.txt, time: 5.040000
/bbs/Gossiping/M.1495290357.A.AAB.html download to ptt/Gossiping/20569_12.txt, time: 5.200000
/bbs/Gossiping/M.1495290361.A.E25.html download to ptt/Gossiping/20569_13.txt, time: 4.940000
/bbs/Gossiping/M.1495290365.A.C5C.html download to ptt/Gossiping/20569_14.txt, time: 5.880000
/bbs/Gossiping/M.1495290406.A.984.html download to ptt/Gossiping/20569_15.txt, time: 6.930000
/bbs/Gossiping/M.1495290448.A.1FF.html download to ptt/Gossiping/20569_16.txt, time: 4.280000
/bbs/Gossiping/M.1495290454.A.FB6.html download to ptt/Gossiping/20569_17.txt, time: 9.880000
/bbs/Gossiping/M.1495290491.A.4D8.htmlTraceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 438, in send
    timeout=timeout
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 649, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
requests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./crawler.py", line 177, in <module>
    main(sys.argv[1:])
  File "./crawler.py", line 158, in main
    downloadForum(forumName, startPage, pages)
  File "./crawler.py", line 140, in downloadForum
    downloadPage(str(articleCount), forumName, pageNum, 'https://www.ptt.cc'+link)
  File "./crawler.py", line 82, in downloadPage
    pageText = getPageText(link)
  File "./crawler.py", line 13, in getPageText
    pageText=requests.get(link, cookies={'over18':'1'}).text
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 488, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
